backend/apps/owasp/graphql/nodes/project_health_metrics.py

```
@@ -0,0 +1,55 @@
+        """Resolve last release age in days."""
+        return self.last_release_days
+
+    @strawberry.field
+    def owasp_page_last_update_days(self) -> int:
+        """Resolve OWASP page last update age in days."""
+        return self.owasp_page_last_update_days
+
+    @strawberry.field
+    def project_name(self) -> str:
+        """Resolve project node."""
+        return self.project.name
```

Pylint (3.3.7)

[error] Instance of 'ProjectHealthMetricsNode' has no 'project' member

(E1101)